### PR TITLE
Update the VF args for OvS / OvS user bridges for nmstate provider

### PR DESCRIPTION
**Pull request by karthiksundaravel**
The default values of the VF parameters like Trust, Spoofcheck, Promisc are updated for OvS and OvS user bridges, Linux bonds. This update is not effectively applied on the nmstate provider's sriov_vf_data, leading to inconsistency between the sriov map and the VF configurations applied. Now the update of the default values are applied on sriov_vf_data.

Depends-On: https://github.com/os-net-config/os-net-config/pull/40

### CODE REVIEW

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by karthiksundaravel** [comment]
recheck

**Review by vcandapp — CHANGES_REQUESTED** [review]

File: os_net_config/tests/test_impl_nmstate.py [hunk 1]
```diff
@@ -1479,6 +1480,394 @@ def test_get_vf_devname(device, vfid):
...
+            other_config: {'mac-table-size': 50000}
+        """
+
+        vf_config = self.provider.prepare_sriov_vf_config()
+        self.assertEqual(yaml.safe_load(exp_pf_config),
+                         vf_config)
+        self.assertEqual(yaml.safe_load(exp_bridge_config),
+                         self.get_bridge_config('br-bond'))
+
+    def test_sriov_pf_with_custom_nicpart_ovs_user_bridge(self):
```
Comment (karthiksundaravel) [inline os_net_config/tests/test_impl_nmstate.py:1722]: @vcandapp The testcase with custom values for sriov_vf is handled here

**Comment by vcandapp** [comment]
lgtm

**Review by vcandapp — APPROVED** [review]

**Pull request merged**